lib/plugin/allure.js

```diff
@@ -215,7 +215,13 @@ module.exports = (config) => {
     }
 
     err.message = err.message.replace(ansiRegExp(), '');
-    reporter.endCase('failed', err);
+    if (reporter.getCurrentTest()) {
+      reporter.endCase('failed', err);
+    } else if (err) {
+      // this means before suite failed, we should report this.
+      reporter.startCase(`${reporter.getCurrentSuite().name} BeforeSuite failed`);
```

Comment:
how about rephrasing to

```js
reporter.startCase(`BeforeSuite of suite ${${reporter.getCurrentSuite().name} } is failed.`);
```

Reply:
sure, except `is failed` isn't quite correct phrasing, i'll make a change

Reply:
https://github.com/Codeception/CodeceptJS/pull/2248/commits/69c6c960420ec3b32894e1c4bb6afe4cd7b4e23d